charts/fission-core/values.yaml

```diff
@@ -110,11 +110,22 @@ router:
 
 ## Persist data to a persistent volume.
 persistence:
-  ## If true, fission will create/use a Persistent Volume Claim
+  ## If true (and storateType is not set), fission will create/use a Persistent Volume Claim
   ## If false, use emptyDir
   ##
   enabled: true
 
+  ## Must be set only if using S3 backend. If using pvc just leave at `enabled: true`
+  ## If storateType is set, one of its backend configuration must be set as below.
+  # storageType: s3
+
+  ## Sample configruation for AWS s3 storage backend
+  # s3:
+  #   bucketName: <awsBucketName>
```

Comment:
I am myself not convinced - hence I commented on the issue, but a folder within bucket should be an option?

Reply:
Yes. I agree. I can keep this in parallel. ONce other issues are reviewed and sorted, I will add this logic to current working setup ( using buckets).

Reply:
I had a loot at this, but couldn't find if `stow` can create subDir within a bucket. Couldn't find any example in their repo. Also [here](https://github.com/graymeta/stow/blob/master/s3/location.go#L27) it seem they only support creating buckets right now.